agent/betting_orchestrator.py

```
@@ -0,0 +1,463 @@
+        try:
+            # Step 3: Convert USDT0 → USDC
+            conversion_result = self.conversion_service.convert_usdt0_to_usdc(
+                usdt0_amount=bet_amount_usdt0,
+                user_address=user_address,
+            )
+
+            usdc_amount = conversion_result["usdc_amount"]
+            conversion_tx_hash = conversion_result.get("tx_hash")
+
+            # Step 4: Determine order side and price
+            side = "BUY" if outcome.upper() == "YES" else "BUY"
```

Ruff (0.14.10)

Useless `if`-`else` condition

(RUF034)